docs/en/docs/tutorial/path-params.md

```diff
@@ -85,7 +85,7 @@ And when you open your browser at <a href="http://127.0.0.1:8000/docs" class="ex
 
 And because the generated schema is from the <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md" class="external-link" target="_blank">OpenAPI</a> standard, there are many compatible tools.
 
-Because of this, **FastAPI** itself provides an alternative API documentation (using ReDoc):
+Because of this, **FastAPI** itself provides an alternative API documentation (using ReDoc), which you can access at <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>:
```

Comment:
I'm not sure that we can assume the existence of this URL.

Reply:
it was already merged, nvm

Reply:
Yeah, I was thinking that, but the example right above for Swagger UI assumes it's running (from the previous examples), so I guess it makes sense to have the link here too.